frontend/packages/schema-editor/src/components/NodePanel/HeadingRow/HeadingRow.module.css

```diff
@@ -1,23 +1,14 @@
 .root {
-  --gap: var(--fds-spacing-1);
-  align-items: center;
   display: flex;
-  padding: var(--gap) 0;
-  gap: var(--gap);
-}
-
-.heading {
-  display: contents;
+  align-items: center;
+  height: var(--subtoolbar-height);
+  gap: var(--fds-spacing-3);
+  border-bottom: 1px solid var(--fds-semantic-border-neutral-subtle);
 }
 
 .heading .headingButton {
-  border-bottom-left-radius: 0;
-  border-left-color: transparent;
-  border-left-style: solid;
-  border-left-width: var(--studio-treeitem-vertical-line-width);
-  border-top-left-radius: 0;
+  display: flex;
   font: var(--fds-typography-paragraph-short-large);
-  min-height: var(--fds-sizing-12);
```

Comment:
Disse egenskapene er her for å skape en likhet mellom rotelementet og underelementene når de er valgt (se bilder). Hva skjer når du tar dette bort?

![image](https://github.com/user-attachments/assets/cedc5bb1-3cd5-4193-a4c3-726acee3d2cd)
![image](https://github.com/user-attachments/assets/825e932d-8fcf-4abc-9f29-b03c7c0a285c)

Reply:
Oi, det var ikke meningen. Bra du fanget det opp!

Ser at dette må være tilstede:
```css
display: flex;
font: var(--fds-typography-paragraph-short-large);
border-left-width: var(--studio-treeitem-vertical-line-width);
border-top-left-radius: 0;
border-bottom-left-radius: 0;
```

Mens det virker som om dette ikke har noen funksjon:
```css
border-left-color: transparent;
border-left-style: solid;
```

Bør være greit å ta vekk, eller?

Reply:
Jeg tror faktisk de to linjene også må være der. Når det står `border-color: transparent`, er det ofte for å hindre at ting flytter på seg når konturen blir borte eller dukker opp. Her skjer det når brukeren bytter mellom valgt / ikke valgt. 

Eventuelt kunne vi valgt en helt annen fremgangsmåte her, f.eks. med `background` med `linear-gradient` eller med `box-shadow`. Da kan vi unngå denne litt uheldige effekten av å bruke `border`:
<img width="535" alt="image" src="https://github.com/user-attachments/assets/219e5bda-feb7-4f49-8f35-f537416fcb3c">